addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -154,6 +154,9 @@ models.Order = models.Order.extend({
             line.set_dirty(false);
         });
         this.trigger('change',this);
+        if (this !== this.pos.get_order() && this.server_id) {
```

Comment:
```suggestion
        const isTheCurrentOrder = this.pos.get_order() && this.pos.get_order().uid === this.uid;
        if (!isTheCurrentOrder && this.server_id) {
```
As you told me, the orders are invalidated, so the order object is different after fetching the new order details from the server. So to check the similarity of the order to the current order, we compare the uid.

Kindly correct my logic if it's wrong.

Also, can you add a comment like this?
```
We only sync when the order is the current order, because it's the time when the "locally printing resume" will be invalidated without reaching the server. 
```

Reply:
Hmm, absolutely. The comparison by reference would not make sense if the user closes then re-opens the table before the saveChanges() is called. I will change it

Reply:
Hi @caburj . We found out that we needed to compare by reference after all. Even if the user opens the same order. The invalidated changes are still important since the opened order is technically not the same even if it has the same uid. In short, we want to prevent the additional sync only and only if the user does not close the table.

Should I open a new PR? Thanks

Reply:
Let's continue the discussion on this new PR.